src/commands/app/build.js

```diff
@@ -67,7 +67,7 @@ class Build extends BaseCommand {
       if (!flags['skip-static']) {
         if (config.app.hasFrontend && (flags['force-build'] || !fs.existsSync(config.web.distProd))) {
           if (config.app.hasBackend) {
-            const urls = await RuntimeLib.utils.getActionUrls(config)
+            const urls = await RuntimeLib.utils.getActionUrls(config, false, false, true)
```

Comment:
we can add unit tests to validate `getActionUrl` call in called with correct parameters.